portfolio/src/main/webapp/vote.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://www.gstatic.com/charts/loader.js"></script>
+    <script defer src="navbar_loader.js"></script>
+    <script defer src="charts.js"></script>
+  </head>
+  <body>
+    <div id="content">
+      <div id="navbar"></div>
+      <h1>Results</h1>
+      <p>Choose the poll you want to see.</p>
+      <form onchange="drawPoll()" method="POST">
```

Comment:
nit: You don't need this for a form that's never actually submitted

Reply:
Good catch! Fix pusehd.